src/main/java/edu/cornell/kfs/coa/document/validation/impl/CuAccountGlobalRule.java

```diff
@@ -0,0 +1,51 @@
+package edu.cornell.kfs.coa.document.validation.impl;
+
+import java.text.MessageFormat;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.kfs.coa.businessobject.AccountGlobalDetail;
+import org.kuali.kfs.coa.document.validation.impl.AccountGlobalRule;
+import org.kuali.kfs.kns.document.MaintenanceDocument;
+
+import edu.cornell.kfs.sys.CUKFSKeyConstants;
+import edu.cornell.kfs.sys.CUKFSParameterKeyConstants;
+import edu.cornell.kfs.sys.CUKFSPropertyConstants;
+
+@SuppressWarnings("deprecation")
+public class CuAccountGlobalRule extends AccountGlobalRule {
+    
+    @Override
+    protected boolean processCustomRouteDocumentBusinessRules(MaintenanceDocument document) {
+        boolean valid = super.processCustomRouteDocumentBusinessRules(document);
+        if (doesAccountGlobalHaveAccountRestrictionCodeAndNoSubFUndGroupCode()) {
+            valid = isTheNewRestrictionCodeValidForEachAccount() && valid;
+        }
+        return valid;
+    }
+
+    protected boolean doesAccountGlobalHaveAccountRestrictionCodeAndNoSubFUndGroupCode() {
```

Comment:
Extra capital in method name SubFUnd => SubFund

Reply:
ok